### PR TITLE
Allow setting KeyPair from Client object

### PR DESCRIPTION
**Pull request by elliotfehr**
This allows changing the KeyPair in a SecureHttpClient from Client object.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@ vendor/
 .php_cs.cache
 doc/.couscous
 .subsplit
+.idea
```
Comment (tgalopin) [inline .gitignore]: You should put this line in your global Git configuration (see https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration).

**Comment by elliotfehr** [comment]
@tgalopin i've made the changes you requested.  Builds seem to be failing, however they do not appear to be due to any changes requested in the PR or in https://github.com/acmephp/acmephp/pull/73.

**Comment by tgalopin** [comment]
Hello!

Could you rebase on master? I fixed the deprecations that were failing. Thanks!

**Comment by elliotfehr** [comment]
@tgalopin I've pulled in the latest master and all tests are passing now! Thanks!

**Comment by tgalopin** [comment]
Thanks @elliotfehr !

**Pull request merged**